.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Description
+
+> Add a small description of what you are doing
+
+## Issue
+
+> Add a link to the issue if any. Fixes #(issue)
```

Comment:
I try to keep the documentation up-to-date with the code. To do so, almost every developments should be accompanied with a spec update. Most of the time I'll try to create the documentation update commits ahead so that contributors can link to the specific commit but some could bring in ideas of their own.

Could you add a section here to link to another PR/commit with the documentation update ? 

Something like:

```markdown
## Specification

> Add a link to a commit with the update of specification that your commit implements (if your commit implies a change in the documentation). It can be in this PR or another, implemented by you or someone else.

```